### DISCUSSION TITLE
Function to immediately activate drawing mode when Alt-Tabbing to ppInk?

### DISCUSSION THREAD

**Discussion by hced** [pubpub-zz/ppInk#265] [Announcements]
When activating ppInk, is it possible to have it automatically be in drawing mode? If not, that's a setting I'd wish for. It allows me to assign a button on my Stream Deck and instantly start annotating.

**Comment by pubpub-zz** [comment]
you have 3 options:
a) if you pass --StartInking as a parameter it will start immediately
b) if ppink is already loaded, any call to ppink.exe will start inking
c) using the Rest API

you may be interested into this last option as you may also add sm extra command within your stream deck  (selecting a tool/pen...).
